src/main/java/seedu/address/logic/commands/addcommand/AddIssueCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands.addcommand;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+public class AddIssueCommand extends AddCommand {
```

Comment:
Since we are including issues will we also have some sort of ResolveIssueCommand as well? If so, you would need to add that as well here.

Reply:
Oh yea we should add that too. Since only Issues will need resolving, I think we only need one ResolveCommand. What do you guys think?